frontend/public/locales/en/common.json

```diff
@@ -271,6 +271,24 @@
   "login-provider-verification-not-org-member": "You are currently logged in to <1>{{login_provider}}</1> as <3>@{{current_name}}</3>. This account is not a member of the <5>@{{organization}}</5> organization. To verify this app, you must request admin privileges in the organization or sign into an account that already has them.",
   "login-provider-verification-failed": "The verification requirements were not met.",
   "login-provider-verification-user-does-not-exist": "The account <1>@{{required_name}}</1> does not exist on <3>{{login_provider}}</3>. You must claim this account name and use it to verify your app, or ideally, choose an app ID that you have control over. See <5>the Flatpak documentation</5> for information about good app IDs.",
+  "developer-invited": "You have been invited to become a developer",
+  "accept": "Accept",
+  "accept-invite": "Accept Invite",
+  "decline": "Decline",
+  "invite-not-found": "Invite not found",
+  "invite-not-found-description": "The invite you are trying to accept or decline does not exist. It might have been cancelled or declined.",
+  "invited-apps": "Invitations",
+  "invite": "Invite",
+  "invite-code": "Your invite code is <1>{{inviteCode}}</1>. Give this code to another developer to allow them to invite you to their app.",
+  "invite-pending": "Invite pending",
+  "primary-developer": "Primary developer",
+  "invite-developer": "Invite developer",
+  "invite-developer-description": "Enter the developer's invite code to invite them to become a developer of this app. Their developer code is found on the Developers tab of their account page.",
```

Comment:
```suggestion
  "invite-developer-description": "Enter the developers invite code to invite them to become a developer of this app. They can find the code on the Developers tab of their account page.",
```

Reply:
This also makes me wonder, if we should inject the translation for `Developers` here, to make sure it's not translated differently

Reply:
I think "developer's" is correct? Fixed the others.

Reply:
Reading up on this, I think you're right